README.md

```diff
@@ -70,18 +70,12 @@ After cloning the repo, run `npm install` to install dependencies.
 
 Run `npm run build` once, or any time icon or language files are changed.
 
-If you don't have it already, install the [Polymer CLI](https://www.polymer-project.org/3.0/docs/tools/polymer-cli) globally:
-
-```shell
-npm install -g polymer-cli
-```
-
 ### Running the demos
 
-To start a [local web server](https://www.polymer-project.org/3.0/docs/tools/polymer-cli-commands#serve) that hosts the demo page and tests:
+To start a local web server that hosts the demo page and tests:
```

Comment:
Would it still be helpful to have [a link to the server](https://open-wc.org/developing/es-dev-server.html#startserver) in here?

Reply:
Good call, done.